styleguide/components/01-mixins/themify/_themify.scss

```diff
@@ -18,13 +106,61 @@ $themes: map-merge($themes, $overwrite-themes);
 //   The exception value for the themified property.
 //   This is the exception optionally used in subthemes.
 //
+//
 @function set-themified-property($colors, $default, $exception) {
   @if map-has-key($colors, $exception) {
     @return map-get($colors, $exception);
   }
   @else {
```

Comment:
No need for else, already returned.

Reply:
This is needed because if the $exception is not found it needs to calculate the other color based on the given $default.